survey/src/main/java/com.quickbirdstudios.surveykit/AnswerFormat.kt

```diff
@@ -29,46 +35,120 @@ sealed class AnswerFormat {
         @StringRes val maximumValueDescription: Int,
         @StringRes val minimumValueDescription: Int
     ) : AnswerFormat() {
-        enum class Orientation { Horizontal, Vertical; }
+        // TODO when creating the scale part anew, also create Vertical orientation
```

Comment:
No, this is just a reminder, IF we create the ScalePart anew, that we support both orientations... for now there is just one, since the default ScaleView of android does not have a rotation option.